charger/phoenix-ev-eth.go

```diff
@@ -0,0 +1,205 @@
+package charger
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/andig/evcc/api"
+	"github.com/andig/evcc/util"
+	"github.com/andig/evcc/util/modbus"
+	"github.com/volkszaehler/mbmd/meters/rs485"
+)
+
+const (
+	phxEVEthRegStatus     = 100 // Input
+	phxEVEthRegChargeTime = 102 // Input
+	phxEVEthRegMaxCurrent = 300 // Holding
+	phxEVEthRegEnable     = 400 // Coil
+
+	phxEVEthRegPower  = 120 // power reading
+	phxEVEthRegEnergy = 904 // energy reading, 128 for fw <= 1.11
+
+	phxEVEthRegPowerScale   = 364 // power scaler
+	phxEVEthRegEnergyScale  = 372 // energy scaler
+	phxEVEthRegCurrentScale = 902 // current scaler, 358 for fw <= 1.11
```

Comment:
Von den realen Registernwerten her könnte es tatsächlich in dem Bereich was sein aber irgendwie passt das überhaupt nicht zur Doku...

![grafik](https://user-images.githubusercontent.com/4681172/109725646-bd486c00-7bb1-11eb-8bf9-b1ee8ba8b4dd.png)



In 902 steht dann 34464 drin. Das passt nicht.

Reply:
Der Scaler bei der grossen ist float, kann schon sein